### DISCUSSION TITLE
Pressed Opacity Acts as 100%

### DISCUSSION THREAD

**Discussion by ghost** [SFX-WoW/Masque#208] [Issues] [answered]
Hi there, cheers for this great AddOn over all these years.

I'm writting here because Curse website comments are bugged and I'm unable to write there. I wanted to ask if there's going to be any update, as for 8.3 I'm having some issues with the AddOn, mainly the "Pressed" button opacity not working. If I turn it all the way down to 0%, I still get 100% colour when pressing the buttons, whatever I do. I just hate the "spam" glow border when I press spells and would like to be able to remove them as before.

I'm using this with ElvUI, but ElvUI doesn't have any button colouring options so I don't think it overrides it (as every other Colour option from Masque works fine). Tried it without any AddOn and get the same issue.

Cheers and stay safe.

**Comment by StormFX** [comment]
I'm not sure what you mean by "spam glow border". Using the Default or Classic skin, the Pushed texture is the gold border around the button's edge when it's pushed. It does not "spam" and only appears when it's in the "pressed" state (ie, holding down a mouse button while hovering it). Switch to either of those skins and make sure that you're adjusting the correct texture. Also, make sure "AddonSkins" is disabled in _ElvUI_. It's not compatible with _Masque_.

I checked with _Dominos_ and it works fine so it's possible that _ElvUI_ or whatever action bar add-on you're using is changing things after _Masque_ has applied the skin, in which case you'll have to submit a ticket to that project.

As far as an 8.3 update goes, the only thing it currently needs is a ToC bump, which doesn't justify a build. Once the translations are more updated, I'll push a build then.

**Comment by ghost** [comment]
By "spam glowing border" I mean that without Masque options, when you press a spell, you see a glow around the button. With Masque, before, I used to be able to lower the opacity of that glow to 0%, so when I spam my buttons while doing my rotations, I wouldn't see the "glowing border" spam over the buttons.

Problem is, with any AddOns active or not, Masque in my case works fine in every other aspect, but the Opacity for "Pressed Button" doesn't work. I had ElvUI and AddonSkins + any other AddOns disabled and still doesn't work.

[Here's a picture showing the big spell in the middle being pressed](https://imgur.com/siwuq8r), where I've chosen red colour to make it more easy to see. Even at 0% opacity, when the button is pressed, I still get 100% opacity.

Because all I want is to get rid of the glowing border when pressing the button, just like I used to.

**Comment by StormFX** [comment]
It works fine with Dominos, which reuses default action bars. I'm not sure what's going on on your end. Just out of curiosity, what skin are you using?

**Comment by ghost** [comment]
I'm using "Dream", although same tests were done with the default ones and I get the same issue.

Also, don't know if it's intended but with all AddOns off and just Masque on, Masque won't work with Blizzard default action bars.

**Comment by StormFX** [comment]
What action bar are you using? Also, _Masque_ doesn't work with the default UI. It's literally in the description. :p

Not sure if it'll have any effect, but to be sure, try resetting the skin and then reloading the UI.

**Comment by ghost** [comment]
If by "resetting" you mean the "Colour Reset" under the skin's colour options, then... still doesn't work.

To be honest, I'm out of ideas. And it's really something that bothers me a lot, as I've started using Masque back in the days just because of this specific option.

Right now I only use ElvUI action bars. In theory, even without AddonSkins, it should work just fine, right? I'll give it a go with Dominos, but would be great if it worked straight with ElvUI. Also, is there a reason why Masque it's not compatible with Masque?

Thanks again and sorry for all the trouble.

**Comment by StormFX** [comment]
In theory, it should work. But I've no control over how add-ons implement their action buttons or how they interact with _Masque_. Something about ElvUI's action bars is unsetting _Masque_'s settings.

**Comment by ghost** [comment]
Apparently Masque is supported by ElvUI... I guess I'll have to ask them what's the issue.

Because it's very odd that just that bit of configuration doesn't work out, every other type of opacity works just fine... Thanks a lot for the quick replies and for trying to solve this up. I'll update this if I get to find out what's the issue or the solution, might help someone else.

Cheers!

**Comment by ghost** [comment]
Okey, so with Dominos works just fine (although I can't see any option to turn off the green glowing border around equipped items on action bars) and I haven't got any answer from ElvUI so far regarding the bug with Masque, so... I guess that's it.

Is there any way to turn off the green border around equiped items? If it is, I'll have to disable ElvUI action bars and just use Dominos instead.

**Comment by StormFX** [comment, marked as answer]
You'll have to check within Dominos's options. I *think* there's an option in there, but not sure. Otherwise, just change the opacity to 0.
